### PR TITLE
Add LS node_stats metricset with event stats

### PR DESCRIPTION
**Pull request by ruflin**
This adds the node_stats metricset to the LS module in Metricbeat with the basic events stats.

### CODE REVIEW

**Comment by tsg** [comment]
I think this doesn't need `needs_backport`. Master still means it's going to be in 6.1.

File: metricbeat/module/logstash/node_stats/node_stats.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+// MetricSet type defines all fields of the MetricSet
+type MetricSet struct {
+	mb.BaseMetricSet
+	http *helper.HTTP
+}
+
+// New create a new instance of the MetricSet
+func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
+	cfgwarn.Experimental("The logstash node_stats metricset is experimental")
```
Comment (ph) [inline metricbeat/module/logstash/node_stats/node_stats.go]: With the release of LS 6.0 which support multiple pipeline, we should also tell them that we are getting global stats for the event rate.

**Comment by ph** [comment]
@ruflin @tsg LGTM, in a future PR we can add easy win stats like HOST + version :)

File: metricbeat/module/logstash/node_stats/node_stats.go (hunk 1 above)
Comment (ruflin) [reply metricbeat/module/logstash/node_stats/node_stats.go]: Lots of additional metricset options here ;-)

**Comment by ruflin** [comment]
Rebased on master with https://github.com/elastic/beats/pull/5639 in and pushed again.

**Comment by tsg** [comment]
Merged, the test failure on Travis was in the Kibana module.

**Pull request merged**